crates/core/src/test_utils/deploy_helpers.rs

```diff
@@ -681,6 +682,7 @@ impl KakarotTestEnvironmentContext {
 
         let starknet_provider = Arc::new(JsonRpcClient::new(HttpTransport::new(sequencer.url())));
         let starknet_account = sequencer.account();
+        jsonrpsee::tracing::info!("Starknet account nonce at start: {:?}", starknet_account.get_nonce().await);
```

Comment:
nice logging throughout this pr

Reply:
this logging was mostly removed and was pushed to debug the current error in Katana.